src/app/chat/views/message_format.nim

```diff
@@ -0,0 +1,35 @@
+proc sectionIdentifier(message: Message): string =
+  result = message.fromAuthor
+  # Force section change, because group status messages are sent with the
+  # same fromAuthor, and ends up causing the header to not be shown
+  if message.contentType == ContentType.Group:
+    result = "GroupChatMessage"
+
+
+
+# See render-inline in status-react/src/status_im/ui/screens/chat/message/message.cljs
+proc renderInline(elem: TextItem): string =
```

Comment:
should all this be in QML instead? given it's very much a presentation thing

Reply:
It would be nice but I'm not sure how to do that. Should this be done with a QObject that represents the `parsedText` attribute of a `Message`? For me it seems that moving this to the view layer would end up increasing the amount of nim code